packages/editor/src/dataviews/actions/export-pattern.tsx

```diff
@@ -22,7 +22,7 @@ function getJsonFromItem( item: Pattern ) {
 		{
 			__file: item.type,
 			title: getItemTitle( item ),
-			content: item.content.raw,
+			content: item.content?.raw || item.content,
```

Comment:
What's safer? 

Maybe a typecheck? E.g., `typeof item.content === 'string' ? item.content : item.content?.raw`

`item.content?.raw` is the expected format of a WordPress post record.

Reply:
Should we update the `Pattern` type to clarify that there are cases where the content is actually a string?